src/cli.js

```diff
@@ -238,9 +238,7 @@ function runCommandFromCli(args: options.Args, flags: options.Flags) {
     return commands.pack(commands.toPackOptions(commandArgs, flags));
   } else if (commandMap.PROJECT[command]) {
     if (commandMap.PROJECT_ADD[subCommand]) {
-      return commands.projectAdd(
-        commands.toProjectAddOptions(subCommandArgs, flags)
-      );
+      return commands.projectAdd(commands.toAddOptions(subCommandArgs, flags));
```

Comment:
We should keep using `toProjectAddOptions` , it won't add a lot of code but keep structure simple to follow

Reply:
Agree, I was thinking can this be moved to some other util folder or what?

Reply:
Also for testing we can create common folder to stow all the common things that we have.